test/Foundatio.TestHarness/Foundatio.TestHarness.csproj

```diff
@@ -1,14 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.5;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard1.5;netcoreapp1.1;net46</TargetFrameworks>
```

Comment:
The pr looks good, we might have to revert the netcoreapp1.1 later but it's all good.